optimum/bettertransformer/models/distilbert/modeling_distilbert_fast.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2022 The HuggingFace and Meta Team.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import torch
+import torch.nn as nn
+
+
+class DistilBertLayerFast(nn.Module):
+    def __init__(self, bert_layer):
+        r"""
+        A simple conversion of the Distill-BERTLayer to its `Fast` implementation.
```

Comment:
Same question about the layer name?
Are you refering to the actual class or the basic layer of that model? Should be consistent for all the `XXXLayerFast`.

Reply:
Should be fixed in 1b87882dc27b9ce9bc6bba7665493c24d9693dae I standardized all the class names there